heroku-server-example.js

```diff
@@ -0,0 +1,14 @@
+var app = require('express')();
+var server = require('http').createServer(app);
+var io = require('socket.io')(server);
+
+app.set('port', (process.env.PORT || 3000));
+
+app.get('/ece926d8c0356205276a45266d361161', function(request, response) {
+    response.send("OK");
+    io.emit('action');
```

Comment:
Good point.  I had these files lying around, but clearly, they weren't in sync.  This is the basic idea, though.  Consider this code untested.